handbook.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: contribute
+title: Handbook
+---
+
+# How we make things
+
+## Branching and SCM
+
+We use git for source control. Our projects in
+[theforeman](https://github.com/theforeman) follow this model for branching:
+
+* **develop**
+    * All pull requests are merged here.
+    * Contains nightlies code. It's relatively stable, but we make no promises about it.
+    * [`test_develop` in Jenkins](http://ci.theforeman.org/job/test_develop/) ensures our tests are passing at all times for various configurations.
+* **release-stable (i.e. 1.10-stable)** -
+    * Code is cherry-picked using `git cherry-pick -m` into this branch.
```

Comment:
It might be worth saying somewhere that individual devs don't need to maintain this?  (it's part of http://projects.theforeman.org/projects/foreman/wiki/Release_Process#Preparing-a-release)